pacemaker_metrics.go

```diff
@@ -237,6 +243,16 @@ func (c *pacemakerCollector) recordFailCountMetrics(pacemakerStatus pacemakerSta
 	}
 }
 
+func (c *pacemakerCollector) recordResourceAgentsChanges(pacemakerStatus pacemakerStatus, ch chan<- prometheus.Metric) {
+	t, err := time.Parse(time.ANSIC, pacemakerStatus.Summary.LastChange.Time)
+	if err != nil {
+		log.Warnln(err)
+		return
+	}
+	// is the resource have changed we set a different timeout from pacemaker
+	ch <- prometheus.NewMetricWithTimestamp(t, prometheus.MustNewConstMetric(c.metrics["ha_cluster_pacemaker_config_last_change"], prometheus.CounterValue, 1))
```

Comment:
instead of 0 , is better to use `1` since it will be more inline with our exporter design with 1 as special value of being present

Reply:
good point!

Reply:
mind that the `c.metrics` lookups should not include the namespace and the subsystem, so in this case it would be `c.metrics["config_last_change"]`

Reply:
good yeah typ;o